docu/SICKAppSpaceCodingStarterKit_Documentation.md

```diff
@@ -667,6 +713,451 @@ Script.serveFunction("CSK_ModuleName.setSelectedInstance", setSelectedInstance)
 
 Besides, there are also functions to dynamically add/reset the number of instances.
 
+### FlowConfig Functions and Events
+
+If a module should be available within the FlowConfig, there are some events and function it should provide.
+
+Additionally, there are some other relevant lines of code already mentioned in other chapters. Please search for 'FlowConfig' on this documentation page to find other relevant lines of code.
```

Comment:
if you can find all relevant lines of code here, then you would be more specific and write .." to find all other relevant lines of code."

Reply:
Done